packages/eas-cli/src/project/android/gradleUtils.ts

```diff
@@ -0,0 +1,101 @@
+import { AndroidConfig } from '@expo/config-plugins';
+import fs from 'fs-extra';
+import g2js from 'gradle-to-js/lib/parser';
+
+// represents gradle command
+// e.g. for `:app:buildExampleDebug` -> { moduleName: app, flavor: example, buildType: debug }
+interface GradleCommand {
+  moduleName?: string;
+  flavor?: string;
+  buildType?: string;
+}
+
+interface Config {
+  applicationId?: string;
+  versionCode?: string;
+  versionName?: string;
+}
+
+interface AppBuildGradle {
+  android?: {
+    defaultConfig?: Config;
+    flavorDimensions?: string; // e.g. '"dimension1", "dimension2"'
+    productFlavors?: Record<string, Config>;
+  };
+}
+
+export async function getAppBuildGradleAsync(projectDir: string): Promise<AppBuildGradle> {
+  const buildGradlePath = AndroidConfig.Paths.getAppBuildGradleFilePath(projectDir);
+  const rawBuildGradle = await fs.readFile(buildGradlePath, 'utf8');
+  return await g2js.parseText(rawBuildGradle);
+}
+
+export function resolveConfigValue(
+  buildGradle: AppBuildGradle,
+  field: keyof Config,
+  flavor?: string
+) {
```

Comment:
Missing return type.

Reply:
Can you check if it's easily possible (without too many changes) to turn on the eslint rule to require function return types?